api-doc/api-doc.yaml

```diff
@@ -0,0 +1,7128 @@
+openapi: 3.0.0
+info:
+  title: Elevate-Project
+  version: 1.0.0
+  description: >-
+    The Elevate Project service provides a suite of endpoints for managing various
```

Comment:
@MallanagoudaB Please use this


The Elevate Project service offers a comprehensive suite of endpoints designed to facilitate the creation of projects, both with and without associated programs and solutions. Additionally, it provides the capability to consume project data and generate various reports based on project consumption.